webapp/src/App.tsx

```diff
@@ -30,6 +32,19 @@ const SoundEffectTWO = () => {
   );
 };
 
+const client = createClient({
```

Comment:
Odd this is showing up as a change didn't Arsh add this to Main already? Not a big deal just keep in mind who's version to keep when handling merge/rebases in the future

Main reason is this will show up as a change you made in the "git blame" so if I had questions about it I'd come to you at this point